pallets/parachain-staking/src/weights.rs

```diff
@@ -176,13 +176,16 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 			.saturating_add(T::DbWeight::get().reads(9 as Weight))
 			.saturating_add(T::DbWeight::get().writes(7 as Weight))
 	}
-	fn active_on_initialize(x: u32, y: u32) -> Weight {
-		(10_177_000 as Weight)
-			// Standard Error: 0
-			.saturating_add((24_000 as Weight).saturating_mul(x as Weight))
-			// Standard Error: 0
-			.saturating_add((1_000 as Weight).saturating_mul(y as Weight))
-			.saturating_add(T::DbWeight::get().reads(2 as Weight))
+	fn active_on_initialize(x: u32, y: u32, ) -> Weight {
```

Comment:
```suggestion
	fn active_on_initialize(x: u32, y: u32) -> Weight {
```
did you run `cargo fmt` on this directory?

Reply:
Just did (9be2212)